plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererConstants.kt

```diff
@@ -134,6 +134,8 @@ object CodeWhispererConstants {
     }
     object CrossFile {
         const val CHUNK_SIZE = 60
+        const val NUMBER_OF_LINE_IN_CHUNK = 10
+        const val NUMBER_OF_CHUNK_TO_FETCH = 3
```

Comment:
should we revert these two as well if they're not used

Reply:
they are used